tsconfig.json

```diff
@@ -22,5 +22,5 @@
   },
   "include": [
     "src"
-  ]
+, "src/styles/AuthButton.tsx"  ]
```

Comment:
@Danneytrieu why is this included in the tsconfig file?

Reply:
i have no idea how that ended up in there, i will remove it